apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```diff
@@ -0,0 +1,351 @@
+import { Component, ElementRef, Inject, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import FroalaEditor from 'froala-editor';
+import { ReplaySubject, takeUntil } from 'rxjs';
+import Tribute from 'tributejs';
+import { CustomEmailComponentStore } from './utility/template-froala.store';
+import { MAT_DIALOG_DATA, MatDialog } from '@angular/material/dialog';
+import 'froala-editor/js/plugins.pkgd.min.js';
+import 'froala-editor/js/froala_editor.pkgd.min.js';
+import { GeneralService } from '../../services/general.service';
+@Component({
+    selector: 'template-froala',
+    templateUrl: './template-froala.component.html',
+    styleUrls: ['./template-froala.component.scss'],
+    providers: [CustomEmailComponentStore]
+})
+export class TemplateFroalaComponent implements OnInit {
+    /** Instance of subject input field */
+    @ViewChild('subjectInputField', { static: false }) subjectInputField: ElementRef;
+    /* Aside pane state*/
+    public asideMenuState: string = 'out';
+    /* This will hold common JSON data */
+    public commonLocaleData: any = {};
+    /* This will hold locale JSON data */
+    public localeData: any = {};
+    /* Observable to unsubscribe all the store listeners to avoid memory leaks */
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    /** Holds Store update email template API success state as observable*/
+    public updateCustomEmailIsSuccess$ = this.componentStore.select(state => state.updateCustomEmailIsSuccess);
```

Comment:
```suggestion
    public updateCustomEmailIsSuccess$: Observable<any> = this.componentStore.select(state => state.updateCustomEmailIsSuccess);
```

Add Type in all variables